x-pack/plugins/security/server/authentication/providers/kerberos.ts

```diff
@@ -36,6 +36,15 @@ export class KerberosAuthenticationProvider extends BaseAuthenticationProvider {
    */
   static readonly type = 'kerberos';
 
+  /**
+   * Performs initial login request.
+   * @param request Request instance.
+   */
+  public async login(request: KibanaRequest) {
```

Comment:
__note:__ didn't test this scenario. Will test later to make sure it works as expected.

Reply:
Struggling to setup environment for some reason, but technically it works now :slightly_smiling_face: Will make sure to check that it actually works before merging though.

Reply:
Kerberos Login Selector integration tests are passing, we should be good.